Proton/ProtonCore/ProtonCore.h

```diff
@@ -0,0 +1,19 @@
+//
+//  ProtonCore.h
+//  ProtonCore
+//
+//  Created by Rajdeep Kwatra on 21/3/21.
+//  Copyright © 2021 Rajdeep Kwatra. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+//! Project version number for ProtonCore.
+FOUNDATION_EXPORT double ProtonCoreVersionNumber;
+
+//! Project version string for ProtonCore.
+FOUNDATION_EXPORT const unsigned char ProtonCoreVersionString[];
+
+// In this header, you should import all the public headers of your framework using statements like #import <ProtonCore/PublicHeader.h>
```

Comment:
What about adding headers here?
It fixes these Xcode warnings

<img width="469" alt="image" src="https://user-images.githubusercontent.com/53595/112118366-66faa600-8c10-11eb-98db-4c07bb4253d7.png">

```suggestion
#import <ProtonCore/PRTextStorage.h>
#import <ProtonCore/PREditorContentName.h>
```